src/functionalTest/java/uk/gov/hmcts/reform/userprofileapi/UpdateUserProfileFuncTest.java

```diff
@@ -32,6 +33,30 @@ public void should_create_Update_profile_and_return_200() throws Exception {
         updateUserProfile(updateUserProfileData(), createdResource.getIdamId(), HttpStatus.OK);
     }
 
+    @Test
+    public void should_throw_400_when_Update_profile_with_invalid_first_name() throws Exception {
+        UpdateUserProfileData updateUserProfileData = updateUserProfileData();
+        updateUserProfileData.setFirstName("<This!Is$Invalid");
+
+        updateUserProfile(updateUserProfileData, createdResource.getIdamId(), HttpStatus.BAD_REQUEST);
+
+        updateUserProfileData.setFirstName("%3cscript%3ealert(%22WXSS%22)%3c%2fscript%3e");
+
+        updateUserProfile(updateUserProfileData, createdResource.getIdamId(), HttpStatus.BAD_REQUEST);
+    }
+
+    @Test
+    public void should_throw_400_when_Update_profile_with_invalid_last_name() throws Exception {
+        UpdateUserProfileData updateUserProfileData = updateUserProfileData();
+        updateUserProfileData.setLastName("<This!Is$Invalid");
+
+        updateUserProfile(updateUserProfileData, createdResource.getIdamId(), HttpStatus.BAD_REQUEST);
+
```

Comment:
add verify method in junit test cases

Reply:
for the JUnit's I am not calling a method, I am validating the object before it is used and then it fails validation